package.json

```diff
@@ -9,14 +9,28 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "commander": "^9.1.0",
```

Comment:
Should we pin deps? (We do so in web but doesn't seem to be the standard across repos)